cw-orch-daemon/src/queriers/cosmwasm.rs

```diff
@@ -81,18 +81,14 @@ impl CosmWasm {
         &self,
         address: impl Into<String>,
         pagination: Option<PageRequest>,
-    ) -> Result<Vec<cosmos_modules::cosmwasm::Model>, DaemonError> {
+    ) -> Result<cosmos_modules::cosmwasm::QueryAllContractStateResponse, DaemonError> {
```

Comment:
If we're changing the response here so that the user is aware of the pagination, we should do it everywhere (most of the queriers that accept pagination "unwrap" the sub response).

Reply:
I guess this change was aiming to unify Cosmawm::* functions responses.

Reply:
Yes. This is needed if you can't query the whole contract state at once (for instance if you have a lot of state). 
I agree that pagination should always be reachable in queries, this reassures developpers. @CyberHoward do you agree ?

Reply:
I agree but from my memory the `Response` struct only contained the inner `Vec<...>` so there's no information loss in unwrapping it. If there is then we should return the `Response` itself for sure.